.github/issue_reporting_guide.md

```diff
@@ -74,7 +74,7 @@ docker buildx inspect
 If you also want to check if the issue still appears in the `master` development branch you can run:
 
 ```bash
-docker buildx build --load -t moby/buildkit:dev "https://github.com/moby/buildkit.git"
+docker buildx build --load -t moby/buildkit:dev "https://github.com/moby/buildkit.git" --build-arg BUILDKIT_CONTEXT_KEEP_GIT_DIR=1
```

Comment:
Wonder is we should look at a bake target as follow-up to just do smth like:

```
docker buildx bake https://github.com/moby/buildkit.git image
```

Reply:
That would indeed be nice; not just shorter, but it would encapsulate any other required future build args.  I just came across it as a normal end-user, working my way up to potentially filing an actual bug report.